osm2streets/src/marking.rs

```diff
@@ -1,46 +1,117 @@
+// We use geom and stay in map space. Output is done in latlon.
```

Comment:
Just noting, the git diff is weird here. This file doesn't exist in `main`. And the PR is showing a merge conflict? The origin of this file is mysterious

Reply:
Yes, i know what the wierdness is but haven't merged to fix it yet.